### PR TITLE
updated sidebar to be less intrusive on mobile/thin browsing windows.

### LINKED ISSUE #63: Speech appears on right side on mobile
On smaller screens (aka mobile), it can be useful to disable iframe because you don't have much room anyways. But speech appears in that column and makes everything else narrower.

### PR DESCRIPTION
**Pull request by Muggyfox1**
Sidebar will update if window resizes

Portrait mode (Height > Width) = no sidebar and speechbox fixed on bottom right of window.

Landscape mode (Width > Height) = sidebar as normal.

Resolves #63

### CODE REVIEW

File: autoread.js [hunk 1]
```diff
@@ -113,4 +115,21 @@ function handleCheckbox(currentLine){
 
 function handleSublist(currentLine){
 	return Array.of(...currentLine.childNodes).filter(x=>x.nodeName != "OL" && x.nodeName != "UL").map(x=>x.textContent).join('');
+}
+
+function orientationCheck(){
+		let sBar = document.getElementById("sidebar");
+		let sBox = document.getElementById("speechbox");
+		
+		if(window.innerHeight + 100 > window.innerWidth){
```
Comment (MichaelEbert) [inline autoread.js:124]: I love the condition of "portrait vs landscape". I think that'll work nicely.

File: autoread.js [hunk 2]
```diff
@@ -113,4 +115,21 @@ function handleCheckbox(currentLine){
...
+			//portrait mode
+			sBar.style.position="fixed";
+			sBar.style.bottom="1em";
+			sBox.style.border="2px solid black";
```
Comment (MichaelEbert) [inline autoread.js:128]: I don't love changing the sidebar style here. I'm not sure what to do about it, though. Brainstorming ideas (may be bad):
-Maybe this swaps out the sidebar class (styled in css document) from "sideways" to "bottom"?
-maybe remove speech box from sidebar and leave it floating?

File: autoread.js (hunk 1 above)
Comment (MichaelEbert) [reply autoread.js:124]: what about ipad? do we want that to be considered "portrait" and have this applied?

**Review by MichaelEbert — CHANGES_REQUESTED** [review]

**Comment by MichaelEbert** [comment]
so theres actually abunch of stuff that i'm not sure how to include on mobile. Do you want to play around with CSS and the document structure to get some possiblities for including these in a unified manner?
"extra things" that we need to think about:
- Table of Contents
- "viewing remote" banner / cancel button
- speech box
- iframe/secondary pages
- navigation bar

File: autoread.js (hunk 1 above)
Comment (Muggyfox1) [reply autoread.js:124]: Maybe a manual override button or setting?

**Comment by Muggyfox1** [comment]
closing out this PR, since PR #66 covers this better.

**Pull request closed without merging**